### ISSUE TITLE
Date Field in Document Template

### ISSUE THREAD

**Issue by bakerstu** [nmradcc/documents#24]
Currently the template document uses a SaveDate document property for all of the dates. This is nice in that it is automatically updated with each save.

However, it would likely be better to use a custom document property for the date that is manually updated. In this way, revision dates can be preserved to reflect the time of approval (by voting and/or by NMRA board).

Need to decide what to do here.

**Comment by JuettM** [comment]
The date property field is acceptable to me. I have not yet found how to do it. I have not looked deep into it yet.

**Comment by bakerstu** [comment]
I have updated the template document to use the "Published Date" property field. I've also updated the TR-1-2020 Draft with instructions on how to use this, specifically in section 3.1 provides details. These changes have been pushed to master and also merged to the S-9.1.1 branch.

When starting from the template, everything is setup and this is easy to use. However, the existing documents need to be modified manually, and it is a somewhat complex effort. Since I have experience doing it, and I can make the change quickly, I suggest I be the one to make the updates.
